bin/cli.js

```diff
@@ -8,13 +8,28 @@ const common = require('metarhia-common');
 
 const metaschema = require('..');
 
-const cwd = process.cwd();
-const apiFile = path.resolve(cwd, process.argv[2]);
+let apiFile = path.resolve(process.cwd(), process.argv[2]);
+const packageFile = path.join(apiFile, 'package.json');
+
+if (fs.statSync(apiFile).isDirectory() && fs.existsSync(packageFile)) {
+  apiFile = path.join(apiFile, JSON.parse(fs.readFileSync(packageFile)).main);
+}
 
 try {
   const imports = require(apiFile);
   const namespace = path.basename(apiFile, '.js');
-  const inventory = metaschema.introspect({ [namespace]: imports });
+  let data = '';
+  for (const module in require.cache) {
+    if (!module.includes('node_modules') &&
+      module !== __filename &&
+      fs.existsSync(module)) {
+      data += fs.readFileSync(module, 'utf8') + '\n';
```

Comment:
It can be a binary file in case you are running it on a package with the native addons.
To avoid loading binary addons here, you can probably skip the files with `.node` extension.